articles/setup-ssh-ubuntu-vm-aws/index.md

```diff
@@ -0,0 +1,120 @@
+![Select Instance Type](/engineering-education/setup-ssh-ubuntu-vm-aws/instance_type.png)
+7. Click the button labeled "Next: Configure Instance Details".
+8. On the next three subsequent pages, you do not need to make any changes. (Click the buttons labelled "Next: Add storage", "Next: Add Tags" and "Configure Security Group" respectively.)
+9. On the Configure Security Group page, select create a new security group and leave everything else as is. (Ignore the displayed warning.) Click "Review and Launch".
+![Configure Security Group](/engineering-education/setup-ssh-ubuntu-vm-aws/configure_security_group.png)
+10. On the Launch Page, click on "Launch", which will trigger a prompt. Select "create a new key pair". Name the key pair and download it. (Later on, you will use your_pem_file.pem to SSH into the Virtual Machine instance you are about to launch.) When done, you can then click on "Launch Instances".
+![Launch Instance](/engineering-education/setup-ssh-ubuntu-vm-aws/launch_instance.png)
+
+**CONGRATULATIONS!!** You have managed to set up and launch your own Virtual Machine Instance.
+Now you can move on to connecting to it remotely via the [SSH protocol](https://www.ssh.com/ssh/protocol/).
+
+### SSH into the Amazon EC2 Virtual Machine
+The operating system of your local computer determines the options available to SSH from your local computer to your Virtual Machine instance running Linux (Ubuntu 18.04).
+
+Moving forward, the tutorial assumes your local computer operating system is Linux or macOS X. If this isn't the case, see [Amazon's guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstances.html?icmpid=docs_ec2_console).
+
+First, you need to locate your private key file (your_pem_file.pem) that you downloaded earlier, and move it to the `~/.ssh` directory. To do this, from the terminal in the directory where your_pem_file.pem was downloaded, run:
+
+```bash
+mv your_pem_file.pem ~/.ssh/your_pem_file.pem
+```
+
+Change from your current directory to the `~/.ssh` directory where you placed the private key file:
+
+```bash
+cd ~/.ssh
+```
+
+Next, you need to ensure your private key file (your_pem_file.pem) is not publicly viewable for SSH to work. To do this, run:
+
+```bash
+chmod 400 ubuntu.pem
+```
+
+To gain an understanding of how the `chmod` command works, read up on [Linux File Permissions](https://www.linux.com/training-tutorials/understanding-linux-file-permissions/).
+
+At this point, you're almost ready to SSH into your Virtual Machine Instance. You need one last thing – your Virtual Machine's IP address. You can get this from the AWS console, as shown in the image below.
+![Virtual Machine IP address](/engineering-education/setup-ssh-ubuntu-vm-aws/ip_address.png)<br>
+Now you can ssh into your instance by running the ssh command:
+
+```bash
+ssh -i ~/.ssh/your_pem_file.pem ubuntu@your_virtual_machine_ip_address -v
+```
```

Comment:
You might want to suggest seeing `ssh-add` and `ssh-agent` to add the pem file to the agent.  I say suggest it, and then provide a link below, for example, along side the "For more information on`ssh`, `ssh-add` and `ssh-agent`."

Reply:
I actually didn't know much about the ssh-agent.
Neither its use nor its value.
This actually prompted me to look into it!! And I learnt something new. THANK YOU!! @lcaballero 
I'll most definitely add that and also write about that neat find you helped me stumble upon!